src/pages/iou/IOUCurrencySelection.js

```diff
@@ -167,86 +124,31 @@ class IOUCurrencySelection extends Component {
 
     /**
      * Confirms the selection of currency and sets values in Onyx
-     * @return {void}
+     *
+     * @param {Object} option
+     * @param {String} option.currencyCode
      */
-    confirmCurrencySelection() {
-        IOU.setIOUSelectedCurrency(this.state.toggledCurrencyCode);
+    confirmCurrencySelection(option) {
+        IOU.setIOUSelectedCurrency(option.currencyCode);
         Navigation.goBack();
     }
 
     render() {
         return (
-            <ScreenWrapper onTransitionEnd={() => {
-                if (!this.textInput) {
-                    return;
-                }
-
-                this.textInput.focus();
-            }}
-            >
```

Comment:
Should we be undoing this text input focus? What is this related to?

Reply:
Yeah, this is covered by the `OptionsSelector` via a couple props:

- `autoFocus` is true by default
- `shouldDelayFocus` uses a `setTimeout` to ensure that the text input isn't focused until the page transition is done

So basically we didn't need this here anymore because it's handled by a lower layer.

Reply:
Ok, seems better to do the logic watching the page transition (but maybe a good follow up issue to look into)